inception-imls-opennlp/src/main/java/de/tudarmstadt/ukp/inception/recommendation/imls/opennlp/doccat/OpenNlpDoccatRecommender.java

```diff
@@ -86,12 +87,7 @@ public boolean isReadyForPrediction(RecommenderContext aContext)
     public void train(RecommenderContext aContext, List<CAS> aCasses)
         throws RecommendationException
     {
-        List<DocumentSample> docSamples = extractSamples(aCasses);
-        
-        if (docSamples.size() < 2) {
-            LOG.info("Not enough training data: [{}] items", docSamples.size());
-            return;
-        }
+        List<DocumentSample> nameSamples = extractSamples(aCasses);
```

Comment:
This change modifies the behavior of the code, i.e. is not a refactoring. Or am I wrong?

Reply:
Yes. In order to reduce cognitive complexity, I replaced the large inline loop with two small loops. Doing so can affect functionality and is detrimental to the maintainability of the code, so this change should be removed.